### ISSUE TITLE
all source code wrapped in node.js NativeModule.wrapper: '(function (exports, require, module, __filename, __dirname) { '

### ISSUE THREAD

**Issue by noos** [node-inspector/node-inspector#799]
I think this issue was raised by "eahutchins" on May 29, 2013.
  https://github.com/node-inspector/node-inspector/issues/161

As is, it makes debugging very hard,
and Profiling/heap snapshots also probably won't work.

I'm using:
  node v5.2.0
  Node Inspector v0.12.5
  npm v3.3.12
on WIndows 7.

Thanks.

**Comment by docksteaderluke** [comment]
+1

**Comment by docksteaderluke** [comment]
I don't think this is specifically caused by the inserted require wrapper line since it also appears in Node v4.2.4 but tracing works as expected.

In the meantime, downgrading from Node v5.2.0 to v4.2.4 seems to solve the problem.
